CRM/Contact/Selector.php

```diff
@@ -271,36 +271,37 @@ public static function &links() {
           'qs' => "reset=1&action=update&cid=%%id%%{$searchContext}{$extraParams}",
           'title' => ts('Edit Contact Details'),
           'ref' => 'edit-contact',
-          'weight' => -10,
+          'weight' => CRM_Core_Action::getWeight(CRM_Core_Action::UPDATE),
         ],
       ];
 
       $config = CRM_Core_Config::singleton();
```

Comment:
I think you can remove this unused line now.

Reply:
Is that unused - my head is exploding with the path these go on but I *think* it gets used...

Reply:
Not a blocker. I thought the `$config` variable no longer needed since you're using Civi::settings now.

Reply:
oh that line!